prog/postgres/postgres_server_nexus.rb

```diff
@@ -173,7 +173,7 @@ def before_run
 
     nap 5 if postgres_server.resource.server_cert.nil?
 
-    ca_bundle = postgres_server.resource.ca_certificates
+    ca_bundle = [postgres_server.resource.ca_certificates, postgres_server.resource.trusted_ca_certs].compact.join("\n")
```

Comment:
I'm a bit puzzled by this bundling. The CA used for authenticating clients and the CA used for TLS *can* be the same can also just as well be different. It seems like you intend to make this more a DIY situation for customer (which has the big advantage that they can control the private keying and never give it to us), which would indicate we're doing two distinct things here, right? ssl_ca_file vs ssl_cert_file.